src/user_pins.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+// Include this header in a user peripheral library.
+//
+// A user peripheral library must export a [user_port_init_t x16_user_port_init] to be
+// consumed at runtime by the emulator during initialization of via2. [x16_user_port_init]
+// should return 0 on success, and <0 on error. 
```

Comment:
`true` / `false` or `0` / `-1` would be better than allowing any negative number.